notebooks/tutorial_environment.yml

```diff
@@ -0,0 +1,251 @@
+name: cmp3-tutorial
```

Comment:
Needs to review the list of packages. Only keep the packages that are imported in the notebooks. Do not forget also to add `- python=3.7` which is the python version used by CMP3. Finally, keep only the part specifying the version in the conda dependencies (i.e. `- nilearn=0.8.1`)

Reply:
Done, tested with the new environment